.circleci/config.yml

```diff
@@ -13,18 +13,18 @@ jobs:
       TERM: dumb
     steps:
       - checkout
-      - run: rm -f helm/g2p-sandbox-fynarfin-SIT/Chart.lock helm/g2p-sandbox-fynarfin-SIT/requirements.lock helm/g2p-sandbox-fynarfin-SIT/charts/*
```

Comment:
you cant modify circle ci config like this.
it will affect other deployments as well

Reply:
changed it to defautl. This is just to test the chart.

Reply:
changed it to default. This is just to test the chart.